doc/02-Installation.md

```diff
@@ -0,0 +1,20 @@
+<!-- {% if index %} -->
+# Installing Icinga Web GenericTTS Integration
+
+The recommended way to install Icinga Web GenericTTS Integration
+and its dependencies is to use prebuilt packages for
+all supported platforms from our official release repository.
+Please note that [Icinga Web](https://icinga.com/docs/icinga-web) is required to run Icinga Web GenericTTS Integration
+and if it is not already set up, it is best to do this first.
+
+The following steps will guide you through installing and setting up Icinga Web GenericTTS Integration.
+<!-- {% else %} -->
+<!-- {% if not icingaDocs %} -->
+
+## Installing the Package
+
+If the [repository](https://packages.icinga.com) is not configured yet, please add it first.
+Then use your distribution's package manager to install the `icinga-generictts` package
+or install [from source](02-Installation.md.d/From-Source.md).
+<!-- {% endif %} -->
```

Comment:
This is not shown on the website, the one from above doesn't mention the package name. Are there some steps missing?